test/canaries/roles/otel-dockerized/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Create folder for docker-compose
+  file:
+    path: "/srv/otel_collector/"
+    state: directory
+    mode: '0755'
+- name: copy compose to the machine
+  template:
+    src: ../files/docker-compose.yml.j2
+    dest: "/srv/otel_collector/otel_collector.yml"
+    mode: 0644
+- name: set previous display_name
```

Comment:
What do you think setting `display_name` in the docker-compose template?
Same Ansible variables can be used and we decouple env vars from the role.

Reply:
Do you have an idea how to set it? 🤔